Wikipedia/Code/SavedPageSpotlightManager.swift

```diff
@@ -35,9 +35,10 @@ public extension WMFArticle {
 
         searchableItem.subject = wikidataDescription
         searchableItem.contentDescription = snippet
-        if let imageURL = imageURL(forWidth: WMFImageWidth.medium.rawValue) {
-            searchableItem.thumbnailData = ImageCacheController.shared?.data(withURL: imageURL)?.data
-        }
+        /// SINGLETONTODO
+//        if let imageURL = imageURL(forWidth: WMFImageWidth.medium.rawValue) {
```

Comment:
I didn't dig into what this is actually doing but seems like at the very least we should pull from `MWKDataSource.shared` to assign this data right?